### PR TITLE
Remove empty element return value documentation

### PR DESCRIPTION
**Pull request by xtqqczze**
Empty `returns` documentation tag is inappropriate, it may as well be removed. In some cases the tag is even included on methods with void return value.

* Enable [SA1615](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1615.md): Element return value should be documented.
* Enable [SA1616](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1616.md): Element return value documentation should have text (as suggestion).

### CODE REVIEW

File: .globalconfig [hunk 1]
```diff
@@ -1460,10 +1460,10 @@ dotnet_diagnostic.SA1613.severity = none
...
 # SA1615: Element return value should be documented
-dotnet_diagnostic.SA1615.severity = none
+dotnet_diagnostic.SA1615.severity = suggestion
 
 # SA1616: Element return value documentation should have text
-dotnet_diagnostic.SA1616.severity = none
+dotnet_diagnostic.SA1616.severity = warning
 
 # SA1617: Void return value should not be documented
 dotnet_diagnostic.SA1617.severity = none
```
Comment (iSazonov) [inline .globalconfig:1469]: It is not clear an intention of the PR. Why do we remove the doc tag if we need to fix it?
I'd expect:
dotnet_diagnostic.SA1615.severity = suggestion
dotnet_diagnostic.SA1616.severity =  suggestion
dotnet_diagnostic.SA1617.severity = warning

Comment (xtqqczze) [reply .globalconfig:1469]: No `returns` tag is better then an empty one, I've enabled `SA1617` and updated the PR description.

Comment (xtqqczze) [reply .globalconfig:1469]: We can't enable `SA1617` in this PR, because of the comments issue as expressed in #13978.

Comment (iSazonov) [reply .globalconfig:1469]: It makes no sense to remove doc tags if we should have its and they should have values.

**Comment by xtqqczze** [comment]
PowerShell-CI-windows failures tracked by #14420.

**Pull request closed without merging**